modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc

```diff
@@ -264,7 +264,7 @@ following command:
 
 [source,console]
 ----
-sudo /etc/letsencrypt/<your-domain-name>.sh
+sudo sh /etc/letsencrypt/<your-domain-name>.sh
```

Comment:
Why is this needed? I thought `sudo` would be fine running a shell script directly with an absolute path.